src/pages/premium/index.js

```diff
@@ -0,0 +1,70 @@
+// ** React Imports
+import React, { useState } from 'react'
+
+// ** Next Imports
+import Link from 'next/link'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+
+// ** MUI Components
+import { styled, useTheme } from '@mui/material/styles'
+import { Box, Typography } from '@mui/material'
+
+// ** Configs
+import themeConfig from 'src/configs/themeConfig'
+
+// ** Layout Import
+import BlankLayoutWHeader from 'src/@core/layouts/BlankLayoutWHeader'
+
+// ** Demo Imports
+import Cards from './Cards'
+import Lists from './Lists'
+import FAQ from './FAQ'
+import Testimonials from './Testimonials'
+import Partners from './Partners'
+
+const HeroContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'flex-start'
+}))
+
+const PremiumPage = () => {
+  // ** State
+
+  // ** Hook
+  const theme = useTheme()
+  const router = useRouter()
+
+  // ** Default Country
+  const country = 'in'
+
+  return (
```

Comment:
This value should come from reducer. And set default value to 'in' in the value for country in the reducer.